DEV-5/DEV-5/Plane.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+namespace DEV_5
+{
+    /// <summary>
+    /// Flyable object Plane. Has start speed 200 km/h that change during the flight (+10 km/h for every 10 km of flight)
+    /// </summary>
+    class Plane : IFlyable
+    {
+        private Point coordinates;
+        private readonly int speed = 200;
+
+        public string WhoAmI() => "Plane";        
+
+        public void FlyTo(Point newPoint)
+        {
+            coordinates = newPoint;
+        }
+
+        public double GetFlyTime()
+        {
+            double distance = coordinates.GetDistance(new Point(100, 200, 800));
+            double time = 0;
+            int counter = 0;
+            while (distance > 10)
```

Comment:
please add blank lines between different blocks of code. Example:

string text;

for ()
{

}


return text;